src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -33,6 +33,8 @@
 
     private final StudyService studyService;
 
+    public static final String TMP_LEGACY_DIRECTORY = "11111111-2222-3333-4444-555555555555";
```

Comment:
What is this ?

Reply:
to make the code backward compatible with study manager. so when we create a study from the study manager we could see it in the gridexplore (legacy directory). this is an intermediate step. we will remove this when the study manager will be removed.